.make/protobuf.mk

```diff
@@ -1,3 +1,5 @@
+SHA1_LOCK_FILE := $(abspath $(CURDIR)/../protogen/buf.sha1.lock)
```

Comment:
After running `make generate` in a clean repo I have following untracked changes:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	protogen/buf.sha1.lock
```
Can we just put that on the .gitignore list?

Reply:
I'll add it in the top .gitignore file